### ISSUE TITLE
Unable to Change Overlays While Replay Recording is Paused

### ISSUE THREAD

**Issue by fragger57** [fragger57/creator_overlays#1]
You are unable to change anything about the overlays while the replay recording is paused, but when it is being played you are able to change things just fine.

**Comment by fragger57** [comment]
I've looked into ways to fix this problem, however every solution I think of either doesn't work, or is way beyond my development skills. If anyone has a potential solution to this problem, please let me know.

**Comment by fragger57** [comment]
I've done some testing on things, and I've deduced that the issue is a result of Replay Mod pausing client ticks while the replay recording is paused. This results in the key bind checker not updating, as it uses the END_CLIENT_TICK event to check if a key is pressed. As a result the issue was not having to do with HudRenderCallback like I had initially thought.
I have a fix for this that is a bit unconventional, which is to use HudRenderCallback to check for key presses, rather than END_CLIENT_TICK. It works, but there are some other solutions I want to look into before finalizing this one, as it has some issues associated with it. Mainly, I believe that HudRenderCallback is updated every frame, and ideally I would like to not check for keypresses potentially hundreds of times a second, and also the tracking point movement is super fast now. Like 10x faster than it was before. I've done testing, and there is no meaningful impact on the performance of the game from checking for key presses using HudRenderCallback vs the END_CLIENT_TICK event, so it's a good enough solution to use.

TLDR:
I've figured out the problem. I have a solution that is good enough, but isn't perfect, so I'm still looking into other solutions. This problem will be fixed in the next update!
